CTFd/api/v1/challenges.py

```diff
@@ -201,6 +205,8 @@ def get(self, query_args):
                     "type": challenge_type.name,
                     "name": challenge.name,
                     "value": challenge.value,
+                    "solved": challenge.id in user_solves,
+                    "total_solves": solve_counts.get(challenge.id, 0),
```

Comment:
This key needs to be `"solves"` to match the existing behaviour on the `/challenges/<id>` endpoint. That makes me want to avoid using just `"solved"` for the key above since it's very easy to typo or misread `solved` and `solves`. I'm leaning toward `"solved_by_me"`.

Reply:
I actually like `solved_by_me` a lot